miflora-mqtt-daemon.py

```diff
@@ -303,6 +305,22 @@ def flores_to_openhab_items(flores, reporting_mode):
             if 'device_class' in params:
                 payload['device_class'] = params['device_class']
             mqtt_client.publish('{}/{}_{}/config'.format(topic_path, flora_name, sensor).lower(), json.dumps(payload), 1, True)
+elif reporting_mode == 'wirenboard-mqtt':
```

Comment:
Seems like the `base_topic` is not regarded. Could you please print an informative warning here, if one is defined!? Thanks

Reply:
Just to make sure: did you see this comment?

Reply:
Yes, I saw. I will add warning today.